libraries/standard/mqtt/cbmc/proofs/MQTT_SerializeConnect/Makefile

```diff
@@ -0,0 +1,17 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+HARNESS_ENTRY=harness
+HARNESS_FILE=MQTT_SerializeConnect_harness
+
+DEFINES +=
+INCLUDES +=
+
+REMOVE_FUNCTION_BODY +=
+UNWINDSET += encodeRemainingLength.0:9
```

Comment:
My memory is that even 5 was enough for this loop unwinding, but I haven't read this version of the code.

Reply:
The  RemainingLength to encode will never be larger than 327700, so 3 is enough times to divide by 128 to reach 0 when the loop would end.